client/src/pages/SignUp.js

```diff
@@ -0,0 +1,243 @@
+                    onChange={this.validateChange}
+                    helperText={this.state.usernameError}
+                  />
+                </Box>
+                <Box display="flex">
+                  <TextField
+                    className={classes.textField}
+                    required
+                    name="signupEmail"
+                    label="Email"
+                    placeholder="Enter your e-mail address"
+                    variant="outlined"
+                    error={this.state.emailError !== "" ? true : false}
+                    helperText={this.state.emailError}
+                    onChange={this.validateChange}
+                  />
+                </Box>
+                <Box display="flex">
+                  <TextField
+                    className={classes.textField}
+                    required
+                    type="password"
+                    name="signupPwd"
+                    label="Password"
+                    error={this.state.passwordError !== "" ? true : false}
+                    placeholder="Enter password"
+                    variant="outlined"
+                    helperText="Minimum 6 characters"
+                    onChange={this.validateChange}
+                  />
+                </Box>
+                <Box display="flex">
+                  <TextField
+                    className={classes.textField}
+                    required
+                    type="password"
+                    name="signupPwdConfirm"
+                    label="Confirm Password"
+                    placeholder="Re-enter password"
+                    variant="outlined"
+                    onChange={this.validateChange}
+                    error={
+                      this.state.passwordConfirmError === "Mismatch"
+                        ? true
+                        : false
+                    }
+                    helperText={
+                      this.state.passwordConfirmError === "Mismatch"
+                        ? "Passwords not matching"
+                        : ""
+                    }
+                  />
+                </Box>
+              </form>
+            </CardContent>
+            <CardActions style={{ justifyContent: "center" }}>
+              <Button type="submit" className={classes.button}>
+                Sign Up
+              </Button>
+            </CardActions>
+          </Card>
+        </div>
+        <div className={classes.rightSideContainer}>
+          <div style={{ padding: "25px" }}>
+            <Typography
+              className={classes.whiteText}
+              variant="h6"
+              align="right"
+            >
+              Already Member ?
+              <Button
+                className={`${classes.button} ${classes.marginLeft_2}`}
+                variant="contained"
+                onClick={this.loadSignIn}
+              >
+                Sign In
+              </Button>
```

Comment:
see comment in signin component on this topic

Reply:
Done. Wrapped in "<Link>" now.

Reply:
Done. Wrapped in Link now.